tox.ini

```
@@ -4,9 +4,10 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py37, py38, py39, py310, py311, pypy3, lint
+envlist = py37, py38, py39, py310, py311, pypy3, lint, pkg
 minversion = 3.3.0
 skip_missing_interpreters = true
+isolated_build = true
```

4.0.0 uses isolated builds by default, so might be better to just require that.
```suggestion
minversion = 4.0.0
skip_missing_interpreters = true
```

---

Wow, so that finally changed. Great!